### PR TITLE
Remove unused integration test-package

### PR DESCRIPTION
**Pull request by Ghislain89**
This PR removes an obsolete package that is no longer used.

### CODE REVIEW

File: rush.json [hunk 1]
```diff
@@ -125,7 +119,6 @@
    * LTS versions: https://nodejs.org/en/download/releases/
    */
   "nodeSupportedVersionRange": ">=16.13.0 <17.0.0 || >=18.12.0 <19.0.0",
-
```
Comment (webpro) [inline rush.json:128]: Please don't include unnecessary formatting changes in PRs

**Review by webpro — COMMENTED** [review]
Please don't include formatting changes in PRs. Hard to review + they'll come back with someone's changes in the same file.

File: rush.json (hunk 1 above)
Comment (Ghislain89) [reply rush.json:128]: This is automatically done by rush format as a pre-commit hook due to me changing this file.

**Review by webpro — APPROVED** [review]
✨

**Review by ggobugi27 — APPROVED** [review]
